README.md

```diff
@@ -1,6 +1,6 @@
 # git-herald
 This serverless application will take github events
-(initially new pull requests) and end them to your
+(initially new pull requests) and send them to your
 company's chat channel of choice (initially slack)
```

Comment:
slack is a proper noun and should be capitalized, as in Slack

Reply:
Feel free to open your own PR :trollface: